ctapipe/ml/stereo_combination.py

```diff
@@ -40,8 +47,8 @@ def _weighted_mean_ufunc(tel_values, weights, n_array_events, indices):
 
 class StereoCombiner(Component):
     # TODO: Add quality query (after #1888)
-    algorithm = Unicode().tag(config=True)
-    combine_property = CaselessStrEnum(["energy", "classification", "direction"]).tag(
+    algorithm = List(Unicode()).tag(config=True)
```

Comment:
See my comment below. In any case, we should use the correct plural/singular form of `algorithm(s)`